src/Tickets/Blocks/Ticket/app/editor/template.js

```diff
@@ -52,26 +53,39 @@ class Ticket extends PureComponent {
 			showTicket,
 		} = this.props;
 
-		return (
-			showTicket
-				? (
-					<Fragment>
-						<article className={ classNames(
-							'tribe-editor__ticket',
-							{ 'tribe-editor__ticket--disabled': isDisabled },
-							{ 'tribe-editor__ticket--selected': isSelected },
-							{ 'tribe-editor__ticket--has-tickets-plus': hasTicketsPlus },
-						) }
-						>
-							<TicketContainer clientId={ clientId } isSelected={ isSelected } />
-							<TicketDashboard clientId={ clientId } isSelected={ isSelected } />
-							{ isLoading && <Spinner /> }
-						</article>
-						{ isModalShowing && <MoveModal /> }
-					</Fragment>
-				)
-				: null
-		);
+		return showTicket ? (
+			<Fragment>
+				<article
+					className={classNames(
+						'tribe-editor__ticket',
+						{ 'tribe-editor__ticket--disabled': isDisabled },
+						{ 'tribe-editor__ticket--selected': isSelected },
+						{
+							'tribe-editor__ticket--has-tickets-plus':
+								hasTicketsPlus,
+						},
+						{
+							'tribe-editor__ticket--is-asc': applyFilters(
+								'tribe.editor.ticket.isAsc',
+								false,
+								clientId
+							),
+						}
```

Comment:
@dpanta94 we try to avoid putting integration/feature codes directly into the ET code as much as possible.

So, I would suggest to have the classNames filtered and then add the asc class using the filter.

Reply:
@rafsuntaskin can do that not a problem. WIll change that ;)